helpers/reference-data-reader/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { ethers } from 'ethers'
+import { AggregatorInterfaceFactory } from '@chainlink/contracts/ethers/v0.6/AggregatorInterfaceFactory'
+
+export type ReferenceDataPrice = (
+  contractAddress: string,
+  multiply: number,
+  meta?: Record<string, unknown>,
+) => Promise<number>
+
+export const getLatestAnswer: ReferenceDataPrice = async (
+  contractAddress: string,
+  multiply: number,
+  meta?: Record<string, unknown>,
+): Promise<number> => {
+  if (!meta || !meta.latestAnswer) return getRpcLatestAnswer(contractAddress, multiply)
+
+  return (meta.latestAnswer as number) / multiply
+}
+
+export const getRpcLatestAnswer: ReferenceDataPrice = async (
+  contractAddress: string,
+  multiply: number,
+): Promise<number> => {
+  const rpcUrl = process.env.RPC_URL
```

Comment:
We can use our helper for required env here:

```
import { util } from '@chainlink/ea-bootstrap'
const rpcUrl = util.getRequiredEnv('RPC_URL')
```

As it will fail with a better error, `RequiredEnvError`.